### ISSUE TITLE
load form crashes: if consent SDK is consumed via CocoaPods 

### ISSUE THREAD

**Issue by DKMaverick** [googleads/googleads-consent-sdk-ios#11]
In **PACView.m**, **loadWebView** method tries to obtain bundle url via:
`NSURL *bundleURL =
      [[NSBundle mainBundle] URLForResource:@"PersonalizedAdConsent" withExtension:@"bundle"];`
If host app is generated via CocoaPods, this is nil and results a crash. One possible fix is to replace the above with: `[[NSBundle bundleForClass:[self class]] URLForResource:@"PersonalizedAdConsent" withExtension:@"bundle"]` which would result `$SAMPLE_APP_PATH/Frameworks/PersonalizedAdConsent.framework/PersonalizedAdConsent.bundle/` instead of non-existent `$SAMPLE_APP_PATH/PersonalizedAdConsent.bundle/`
A workaround solution is to change the documentation, so that even with CocoaPods setup, SDK consumers need to add `PersonalizedAdConsent.bundle` to Copy bundle resources manually.

**Comment by shirakaba** [comment]
Thanks for this fix; exactly what I needed. I've made a fork so that people can instantly apply or test this patch if necessary:

```
pod 'PersonalizedAdConsent', :git => 'https://github.com/shirakaba/googleads-consent-sdk-ios.git', :branch => 'fix-cocoapods-bundle-url'
```

I have also made a pull request for DKMaverick's suggestion.

**Comment by rampara** [comment]
This issue should be resolved on the latest version of this SDK (v1.0.1).

**Comment by erkie** [comment]
@rampara it seems that 1.0.1 has yet to be pushed to Cocoapods.org? Thanks!

**Comment by rampara** [comment]
Closing issue as it has been resolved on the latest version of this SDK (v1.0.1).
